Sources/MongoSwift/MongoCollection.swift

```diff
@@ -133,4 +133,97 @@ public class MongoCollection<T: Codable> {
 
         return try body(collection)
     }
+
+    /**
+     * Starts a `ChangeStream` on a collection. By default, the type `CollectionType` is associated with the
```

Comment:
similar to my comment in prev review maybe get rid of the "by default" phrasing and say more like
"The type `CollectionType` will be associated..."